src/Sulu/Bundle/ContentBundle/Admin/CKEditorJsConfig.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Admin;
+
+use Sulu\Bundle\AdminBundle\Admin\JsConfigInterface;
+use Sulu\Component\Security\Authentication\UserInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+/**
+ * Passes the ckeditor toolbar into the js-config.
+ */
+class CKEditorJsConfig implements JsConfigInterface
```

Comment:
Should we consider calling the components 'text-editor' instead of 'ckeditor', as its more generic. Moreover the content-type is called 'text-editor'.